sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineScaleSetImpl.java

```diff
@@ -1798,6 +1820,24 @@ private void setOSDiskDefault() {
         }
     }
 
+    private boolean createFlexibleProfile() {
```

Comment:
isCreateFlexibleProfile

Reply:
Or maybe `shouldCreateProfile`?

Reply:
Yes, it makes sense. This method's purpose is to test if a flexible profile has been created or attached in the current operation(creating flexible vmss or attach a profile to an existing vmss, during both scenarios profile defaults has to be set).
Or `isFlexibleProfileCreated`?